src/csharp/Grpc.Tools/build/_protobuf/Google.Protobuf.Tools.targets

```diff
@@ -271,7 +271,6 @@
       GrpcPluginExe="%(_Protobuf_OutOfDateProto.GrpcPluginExe)"
       GrpcOutputDir="%(_Protobuf_OutOfDateProto.GrpcOutputDir)"
       GrpcOutputOptions="%(_Protobuf_OutOfDateProto._GrpcOutputOptions)"
-      LogStandardErrorAsError="true"
```

Comment:
This is a MSBuild quirk. To use the `LogEventsFromTextOutput` extension point. If this attribute is set, errors will use the default error logging logic which populates a link to the target file instead of the location specified in the error message. When not set, the `LogEventsFromTextOutput` will be called to determine how to handle the output and error messages.